man/sd_journal_get_fd.xml

```diff
@@ -199,22 +199,21 @@ else {
     immediately for all journal changes. Returns 0 if there might be a
     latency involved.</para>
 
-    <para><function>sd_journal_process()</function> and
-    <function>sd_journal_wait()</function> return one of
-    <constant>SD_JOURNAL_NOP</constant>,
-    <constant>SD_JOURNAL_APPEND</constant> or
-    <constant>SD_JOURNAL_INVALIDATE</constant> on success or a
-    negative errno-style error code. If
-    <constant>SD_JOURNAL_NOP</constant> is returned, the journal did
-    not change since the last invocation. If
-    <constant>SD_JOURNAL_APPEND</constant> is returned, new entries
-    have been appended to the end of the journal. If
-    <constant>SD_JOURNAL_INVALIDATE</constant>, journal files were
-    added or removed (possibly due to rotation). In the latter event,
-    live-view UIs should probably refresh their entire display, while
-    in the case of <constant>SD_JOURNAL_APPEND</constant>, it is
-    sufficient to simply continue reading at the previous end of the
-    journal.</para>
+    <para><function>sd_journal_process()</function> and <function>sd_journal_wait()</function> return one of
+    <constant>SD_JOURNAL_NOP</constant>, <constant>SD_JOURNAL_APPEND</constant> or
+    <constant>SD_JOURNAL_INVALIDATE</constant> on success or a negative errno-style error code. If
+    <constant>SD_JOURNAL_NOP</constant> is returned, the journal did not change since the last invocation. If
+    <constant>SD_JOURNAL_APPEND</constant> is returned, new entries have been appended to the end of the journal.  In
+    the case of <constant>SD_JOURNAL_APPEND</constant>, it is sufficient to simply continue reading at the previous end
+    location of the journal, to read the newly added entries. If <constant>SD_JOURNAL_INVALIDATE</constant>, journal
+    files were added to or removed from the current view on the journal stream (possibly due to rotation or vacuuming),
```

Comment:
what is "current view"? I thought INVALIDATE is returned if any journal files in the relevant directories are added or removed, even if they wouldn't be shown because of installed matches.

Reply:
hmm, ok, by "view" i meant SD_JOURNAL_SYSTEM vs. SD_JOURNAL_CURRENT_USER vs. SD_JOURNAL_RUNTIME_ONLY and so on. lemme reword this